src/ruuvitag-adapter.ts

```diff
@@ -157,6 +155,20 @@ export class RuuviTagAdapter extends Adapter {
     this.knownDevices = {};
     addonManager.addAdapter(this);
 
+    // @tim - the manifest that is loaded (I can't work out how) does not have the id
+    //        The id is required to load the configuration, so I hack it in here
+    //        Sometimes I've seen a generated manifest.json in the lib directory
+    manifest.id = manifest.id || 'ruuvitag-adapter';
+
+    const config = getDefaultConfig();
+    const db = new Database(manifest.id);
+    db.open()
+    .then(() => { return db.loadConfig(); })
+    .then((c) => {
+      mergeLoadedConfig(config,c);
+    })
+    .catch((e) => console.error(e));
```

Comment:
There is an even simpler way to do this.
```suggestion
    mergeLoadedConfig(config, manifest.config);
```

Reply:
manifest.config is undefined - I'll investigate if it's there with another name

Reply:
It's there as manifest.moziot.config
Which I think you plan to remove

Reply:
> It's there as manifest.moziot.config

Damn, I forgot the middle part.

> Which I think you plan to remove

What I plan to remove is the `moziot` section in the `package.json`.
Older versions of the gateway (which do not support `setCachedValueAndNotify`) created the `manifest` from the contents of the `package.json`.
Newer gateway versions use the `manifest.json` instead.
In order not to break existing code, the structure remains the same, though.

Reply:
done